### PR TITLE
fix documention on how to run random_agent since python.BUILD was rem…

### PR DESCRIPTION
**Pull request by jifflund**
Opened PR for https://github.com/deepmind/lab2d/issues/8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ random actions. This can be used as a base for creating your own agents, and as
 a simple tool to preview an environment.
 
 ```sh
-bazel run -c opt dmlab2d/python:random_agent -- --level_name=clean_up
+bazel run -c opt dmlab2d/random_agent --config=lua5_2  -- --level_name=clean_up
```
Comment (tkoeppe) [inline README.md]: I would prefer not to prejudice the choice of Lua version here.

If you'd really like to make a note somewhere else, then maybe that could be a separate commit, but note that we don't really support MacOS officially at this point.

Comment (jifflund) [reply README.md]: Thanks for the comment.  Removed the specific version of Lua and kept out references of Mac to keep it simple.
